### PR TITLE
LRQA-40825 | master

### PR DESCRIPTION
**Pull request by kyle-miho**
https://issues.liferay.com/browse/LRQA-40825

### CODE REVIEW

**Comment by anthony-chu** [comment]
ci:test

**Comment by kyle-miho** [comment]
ci:test:sf

**Comment by kyle-miho** [comment]
ci:test:sf

**Comment by kyle-miho** [comment]
ci:test:sf

**Comment by kyle-miho** [comment]
ci:test

File: portal-web/test/functional/com/liferay/portalweb/macros/Button.macro [hunk 1]
```diff
@@ -25,10 +25,23 @@
 		<execute function="AssertClick" locator1="Button#DONE" value1="Done" />
 	</command>
 
+	<command name="clickEditIfPresent">
```
Comment (anthony-chu) [inline portal-web/test/functional/com/liferay/portalweb/macros/Button.macro]: hey @kyle-miho, this macro seems a bit over-protected. in essence, there is no case in which this macro can fail (which we want to happen should a failure condition happen). can you show me later what you're intending to do with this macro?

**Pull request closed without merging**